<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(:/bg.jpg)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1131</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Service Dons</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1131</width>
        <height>591</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <property name="styleSheet">
        <string notr="true">LOOL
</string>
       </property>
       <attribute name="title">
        <string>Donneurs</string>
       </attribute>
       <widget class="QComboBox" name="civilite_p">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Mr</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mme</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="nom_p">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="prenom_p">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>140</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="proffession_p">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>190</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="cin_p">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>250</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="adresse_p">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>310</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="datenaissance_p">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>40</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="email_p">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>160</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>+216</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="tel_p">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>160</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="region_p">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>220</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>40</y>
          <width>581</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Prenom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Profesion</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>260</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Cin</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>320</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>50</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>110</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>170</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Tel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>230</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Region</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter_d">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>380</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier_d">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>380</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer_d">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>380</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="afficher_d">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>380</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Afficher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>500</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
       <widget class="QPushButton" name="quitter_d">
        <property name="geometry">
         <rect>
          <x>1020</x>
          <y>500</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>civilité</string>
        </property>
       </widget>
       <widget class="QPushButton" name="rechercher_d">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>20</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="recherche_d">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>20</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Recherche</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="imprimer_d">
        <property name="geometry">
         <rect>
          <x>1020</x>
          <y>260</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Imprimer</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="fichier_d">
        <property name="geometry">
         <rect>
          <x>1020</x>
          <y>300</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Ouvrir Fichier</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="email_d">
        <property name="geometry">
         <rect>
          <x>1020</x>
          <y>340</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Envoyer E-mail</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="pdf_d">
        <property name="geometry">
         <rect>
          <x>1020</x>
          <y>10</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Pdf</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tri_d">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>430</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri</string>
        </property>
       </widget>
       <widget class="QPushButton" name="afficherstat">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>430</y>
          <width>151</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Statistique</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Dons</string>
       </attribute>
       <widget class="QLineEdit" name="nom_e">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="num_e">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="cin_e">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="tel_e">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>210</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="email_e">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>270</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="date_e">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>50</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="adress_e">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="montant_e">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>200</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_e">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>50</y>
          <width>551</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Num</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Cin</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>210</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Contact</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>280</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>60</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>130</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>210</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Montant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter_1">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>350</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier_1">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>350</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer_1">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>350</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="afficher_1">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>350</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Afficher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="quitter_1">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>500</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>500</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Retour</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="recherche_1">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Recherche</string>
        </property>
       </widget>
       <widget class="QPushButton" name="rechercher_1">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>20</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="imprimer_1">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>240</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Imprimer</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="fichier_1">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>280</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Ouvrir Fichier</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="email_1">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>320</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Envoyer E-mail</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="pdf_1">
        <property name="geometry">
         <rect>
          <x>1010</x>
          <y>20</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Pdf</string>
        </property>
       </widget>
       <widget class="QPushButton" name="statistique">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>410</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Statistique</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tri_1">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>410</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>1</y>
        <width>1131</width>
        <height>591</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Aides</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>321</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt;&quot;&gt;Ce service gére les dons et les donneurs en recuperant les données et les stocker dans une base de donnée.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>250</y>
        <width>185</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Page Facebook</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>185</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Site Web</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
